src/vm/ilmarshalers.cpp

```diff
@@ -2560,6 +2561,12 @@ MarshalerOverrideStatus ILHandleRefMarshaler::ArgumentOverride(NDirectStubLinker
         pcsDispatch->EmitLDC(offsetof(HANDLEREF, m_handle));
```

Comment:
I would be nice to fetch the `m_handle` via `MscorlibBinder::GetField` as well for consistency.